pkgs/development/libraries/xapian/default.nix

```diff
@@ -37,7 +37,5 @@ let
     };
   };
 in {
-  # xapian-ruby needs 1.2.22 as of 2017-05-06
-  xapian_1_2_22 = generic "1.2.22" "0zsji22n0s7cdnbgj0kpil05a6bgm5cfv0mvx12d8ydg7z58g6r6";
```

Comment:
does this need an alias also?

Reply:
The gem was the last consumer for this library.
I considered adding `1.4.9`, since that's the last version of the `xapian-ruby` gem, but until there's anything using it I'd say we can wait for that.